docs/geolocation.md

```diff
@@ -0,0 +1,168 @@
+        if "properties" in geojson_point:
+            properties = geojson_point["properties"]
+            if "strname" in properties:
+                idea.street_name = properties["strname"]
+            if "hsnr" in properties:
+                idea.house_number = properties["hsnr"]
+            if "idea" in properties:
+                idea.zip_code = properties["plz"]
+        idea.save()`
+
+
+`
+        old_point = obj.point
+        # Convert to GEOS Point with correct SRID
+        obj.coordinates = Point(old_point.x, old_point.y, srid=4326)
+        obj.save(update_fields=['coordinates'])
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('a4_candy_mapideas', 'previous_migration'),
+    ]
+
+    operations = [
+        migrations.RunPython(migrate_point_to_coordinates),
+    ]
+```
+
+- Run the migration:
+
+```bash
+python manage.py migrate
+```
+
+#### Step 4: Remove the Old `map_fields.PointField`
+
+- After verifying the new field is populated and working correctly, remove the old `point` field from the model.
+
+- Create and apply a migration to drop the old field.
+
+#### Step 5: Make the New Field Non-nullable (Optional)
+
+- If all data is migrated, update the model to:
+
+```python
+# Before
+coordinates = gis_models.PointField(srid=4326, null=True, blank=True)
+
+# After (rename the field)
+point = gis_models.PointField(srid=4326, null=False, blank=False)
+```
+
+- Create and apply the migration to rename the field.
+
+---
+
+### 3. Important Notes
+
+- **Coordinate Order**: GeoDjango expects `Point(longitude, latitude)`.
+- **Default Values**: When setting defaults for `PointField`, use a `Point` instance, *not* a tuple or list.
+- **Database Setup**: Ensure PostGIS extension is enabled in your database or spatialite3.
+- **Indexing**: Consider adding a spatial index on the new field for efficient spatial queries.
+
+---
+
+### 4. Summary Table
+
+| Step  | Action                             | Notes                                      |
+|-------|------------------------------------|--------------------------------------------|
+| 1     | Add new `coordinates` PointField   | Keep old `point` field temporarily         |
+| 2     | Apply schema migration             | Adds new GIS field to DB                   |
+| 3     | Data migration to copy data        | Use `python manage.py migrate`		  |
+| 4     | Remove old `point` field           | After data verification                    |
+| 5     | Rename new field to point	     | Run `python manage.py makemigrations`      |
+						python manage.py migrate                  |
+
+---
+
+### 5. References
+
+- [Django GeoDjango PointField docs](https://docs.djangoproject.com/en/stable/ref/contrib/gis/model-api/#pointfield)
+- [How to write data migrations](https://docs.djangoproject.com/en/stable/topics/migrations/#data-migrations)
+- [PostGIS setup and spatial indexing](https://postgis.net/docs/manual-3.1/postgis_installation.html)
+
```

Comment:
@partizipation @vellip I added docs for steps to do when we want to migrate old point field to django GIS. So far we have only changed the Project (and the Plan for mB), as it was required for querying and comparing distances for Kiezradar. There is also mapideas, and the budgeting apps, that in the future could migrate to Gis PointField.